bpf/trace_common.h

```diff
@@ -130,7 +131,7 @@ static __always_inline u8 correlated_requests(tp_info_pid_t *tp, tp_info_pid_t *
     // We check for correlated requests which are in order, but from different PIDs
     // Same PID means that we had client port reuse, which might falsely match prior
     // transaction if it happened during the same epoch.
-    if ((tp->tp.ts > existing_tp->tp.ts) && (tp->pid != existing_tp->pid)) {
+    if ((tp->tp.ts >= existing_tp->tp.ts) && (tp->pid != existing_tp->pid)) {
```

Comment:
These could be the same if the requests are too quick to happen one after another.